### ISSUE TITLE
Big padding visible while route is being lalculated

### ISSUE THREAD

**Issue by sonora** [osmandapp/Osmand#9014]
I have somewhere  noted this before  but we never fully  fixed it: 

When you start a route by on the map screen hitting the 'Directions' button, then tap 'Set destination' and use any of our available methods (e.g. select a favorite, use 'Select on map', or use the 'Previos route' button, etc.), this route setup dialog will remain in fully expanded mode with mostly showing just padding, see screenshot. Instead of obscuring the entire map, we should collapse the dialog during route calculation to its medium expansion state, I.e. collapsing the padding between the blue progress indicator and the bottom buttons.

![Screenshot_20200520-084150_OsmAnd~](https://user-images.githubusercontent.com/845735/82414748-d1363d80-9a77-11ea-9693-477cd1569e5d.jpg)

**Comment by vshcherb** [comment]
I think it's done on purpose not to annoy user with animation. So if you make it smaller then map won't be centered to the route after calculation. So there is a logic behind it - full screen will be centered, halfscreen won't.

**Comment by sonora** [comment]
But you can manually swipe it down (e.g. one notch to hide the padding) during the calculation, and there is no adverse effect at all. Just try. (Map will be centered exactly the same after calculation finishes.)

**Comment by vshcherb** [comment]
Even if pan the map ? That must be bug... To be honest I actually don't know the answer yet how it is supposed to be, will check.

**Comment by sonora** [comment]
Yes, I often swipe down thee dialog manually and pan while the route calculates, causes no issue at all.

(Once you start panning, the dialog would minimize all the way, but that's good, gives you more space to pan the map. Map centers immediately to display the the entire route when calculated, no issue at all there.

**Comment by vshcherb** [comment]
It's collapsible and even more collapsed state doesn't allow to move the map, so you will need to close it to browse the map. On the other hand it creates a better screen navigation cause the map position is changing after route is calculated and it shouldn't change if you browse map for example (we shouldn't intervent user input).

So for now, I think to keep it open without any action and think about it. Anyway there are many workarounds even start navigation from context menu to eliminate that step.
